HGGM/Views/Users/Index.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@using Microsoft.AspNetCore.Mvc.Localization
+@model IEnumerable<HGGM.Models.Identity.User>
+@inject IViewLocalizer lang
+
+@{
+    ViewData["Title"] = lang["title"];
+}
+
+<div class="column is-centered">
+    <nav class="panel">
+        <p class="panel-heading">@lang["heading"]</p>
+        <div class="panel-block">
+            <table class="table" style="width: 100%">
+                <colgroup>
+                    <col span="1" style="width: 40%"/>
+                    <col span="1" style="width: auto"/>
+                    <col span="1" style="width: 15%"/>
```

Comment:
`style="width: 15%"` AFAIK these break on mobile devices. If you really need to use them, try them on all widths

Reply:
I commented it out, so you could have a look at it later, i think it looks fine but i don't have your nose for quality :D